missions/CLP_Missions.txt

```diff
@@ -86,31 +90,78 @@ clp_slot1_branch = {
 
 		provinces_to_highlight = {
 			OR = {
-				province_id = 5320
-				province_id = 5322
+				province_id = 5872 # replaced because it felt like it made more geographical sense, willing to take any and all inputs on what provinces should be used
 			}
```

Comment:
If kept at only one province, redundant OR block. Unsure with requested feedback for now, I'll check later.

Reply:
Uusally not an issue